resources/tools/bin/ballerina

```diff
@@ -26,7 +26,11 @@ then
     JAVA_COMMAND=java
     if test -d "$CURRENT_PATH/../dependencies/jdk8u202-b08-jre"; then
       JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u202-b08-jre/bin/java"
+    elif [[ `$JAVA_COMMAND -version 2>&1 | grep -e "1.8."` = "" ]]; then
```

Comment:
shall we not use -e option with grep? There is a known issue with solaris https://github.com/ballerina-platform/ballerina-distribution/issues/359